templates/taxi/manufacturer_list.html

```diff
@@ -23,11 +24,18 @@ <h1>Manufacturer List
               <td>
                   {{ manufacturer.country }}
               </td>
+              <td>
+                <a href="{% url 'taxi:manufacturer-update' pk=manufacturer.id %}">Update</a>
+              </td>
+              <td>
+                <a href="{% url 'taxi:manufacturer-delete' pk=manufacturer.id %}">Delete</a>
```

Comment:
Consider adding a 'Cancel' button next to the 'Delete' link to allow users to navigate back to the previous page, as per the checklist requirement: 'Add `Cancel` button apart from `Delete` one. The `Cancel` button will lead to the previous page the user was on.'